src/node_messaging.cc

```diff
@@ -1167,7 +1196,8 @@ std::unique_ptr<TransferData> JSTransferable::TransferOrClone(
   HandleScope handle_scope(env()->isolate());
   Local<Context> context = env()->isolate()->GetCurrentContext();
   Local<Symbol> method_name = mode == TransferMode::kCloneable ?
-      env()->messaging_clone_symbol() : env()->messaging_transfer_symbol();
+      Symbol::For(env()->isolate(), FIXED_ONE_BYTE_STRING(env()->isolate(),
+          "kClone")) : env()->messaging_transfer_symbol();
```

Comment:
We should probably add these symbols somehow to the `Environment` like we do with strings and other reused objects.

Reply:
Moved them to the `Environment` in https://github.com/nodejs/node/pull/41148/commits/d31188cde9d08121b92995ee4dcfa0e57b002ee6.